agent/handlers/v1/response.go

```diff
@@ -46,7 +46,7 @@ type TasksResponse struct {
 
 // ContainerResponse is the schema for the container response JSON object
 type ContainerResponse struct {
-	DockerID   string
+	DockerID   string `json:"DockerId"`
```

Comment:
same comment as https://github.com/aws/amazon-ecs-agent/pull/1471#discussion_r205248209: 

> I'd also suggest adding unit tests that unmarshal raw strings and validate each of the fields in the unmarshalled object with the expected one

Reply:
Sure, there is a `TODO` in the summary that indicates I will add tests for it.

Will add it soon.

Reply:
@aaithal I would rather use a functional test that consumes the endpoint metadata for this instead of a unit test on the unmarshaller. I think a functional test will raise more warning flags about things like this breaking in the future.

Reply:
I'd vote for both instead of one or the other. You'd know immediately (during development) if you're breaking contracts if you add unit tests.

Reply:
i think in this case either option would have helped because the tests were also changed with the field renames in the original PR. i cant think of any way to really help prevent this other than having more eyes reviewing. lmk if you have any other ideas though. 😕 

i do agree we should add both unit and functional tests though for this change.